### DISCUSSION TITLE
Why Birt 4.8 downloads have been moved to archive

### DISCUSSION THREAD

**Discussion by giupa** [eclipse-birt/birt#1122] [Q&A] [answered]
We are using Birt 4.8 and recently trying to understand an issue we are having I noticed that version 4.8 has been removed from https://download.eclipse.org/birt/downloads/build.php?build=R-R1-4.8.0-201806261756 and after a very long search I could find the files I was looking for in the archive. There is not a redirect URL either.
I would like to know why 4.8 or 4.9 releases are being removed and placed in the archive especially since older releases are still in the main page. see https://download.eclipse.org/birt/downloads/drops/ 
Thanks

**Comment by wimjongman** [comment, marked as answer]
Apologies for the inconvenience. We are in the process of moving all old releases to the archive. The reason is that the old releases contain CVE security risks, and we do not want to serve those to the uninformed user.

If people want to have access to the old releases, they can get them from the [ARCHIVES](https://archive.eclipse.org/birt/downloads/drops/) I will add a link to that on the main page.

**Reply by giupa** [reply]
Thank you Wim, I really appreciate your quick reply.

**Reply by giupa** [reply]
Would you know if is there a list of security vulnerabilities documented per BIRT release and where could we find them ?
Thanks,
Giuliana

**Reply by wimjongman** [reply]
Just google for 'eclipse birt cve'

**Comment by wimjongman** [comment]
A link to the archives was added to the main page: https://eclipse.org/birt
